src/TableColumn/Generic.php

```diff
@@ -10,6 +10,8 @@
 
 /**
  * Implements Column helper for table.
+ *
+ * @property \atk4\ui\Table $owner
```

Comment:
Only a typehint, is it right that this base class will be the owner in 96% of the usages?